Packs/iDefense/Integrations/iDefense_v2/README.md

```diff
@@ -54,9 +55,10 @@ Checks the reputation of the given IP address.
 {
     "DBotScore": {
         "Indicator": "0.0.0.0",
+        "Reliability": "B - Usually reliable",
         "Score": 2,
         "Type": "ip",
-        "Vendor": "iDefense"
+        "Vendor": "ACTI"
```

Comment:
Also, please see lines 100, 203, 235, 267, 306, 309, 312 in the `py` file, where the vendor is still hardcoded as `iDefense`, and change them accordingly.

Reply:
by default it takes value from `id` and `name` attribute of `iDefense_v2.yml` . So it should not be breaking anything. It was previously `iDefense_v2` and still it will be the same.